### PR TITLE
aarch64 PR

### PR DESCRIPTION
**Pull request by step21**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by step21** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -3,11 +3,21 @@ export PATH="$PREFIX/bin:$BUILD_PREFIX/bin:$PATH"
...
+if [ $ARCH == "x86_64" ]; then
+  if [ -f "$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld" ]; then
+     ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld $BUILD_PREFIX/bin/ld
+  fi
+fi
+if [ $ARCH == "aarch64" ]; then
+  if [ -f "$BUILD_PREFIX/bin/aarch64-conda_cos6-linux-gnu-ld" ]; then
+     ln -s $BUILD_PREFIX/bin/aarch64-conda_cos6-linux-gnu-ld $BUILD_PREFIX/bin/ld
+  fi
 fi
```
Comment (isuruf) [inline recipe/build.sh:21]: Haven't tested, but something like below should work.
```suggestion
if [[ -f "$LD" && ! $BUILD_PREFIX/bin/ld ]]; then
   ln -s $LD $BUILD_PREFIX/bin/ld;
fi
```

Comment (step21) [reply recipe/build.sh:21]: yeah, maybe. tbh this was also there already from the original recipe, so not sure how badly it is needed.

**Comment by isuruf** [comment]
@step21, tests are failing. Ping me once the tests pass and I'll merge.

**Comment by step21** [comment]
@isuruf I though I had taken out ppc64le again, but apparently only locally. I was testing it on CI, but turns out it needs a much more extensive build as upstream does not provide a tarball for it, so everything has to be built from scratch and an existing ghc from yum or similar used for bootstrapping, so I decided to take it out, but apparently forgot to push.

**Comment by isuruf** [comment]
You can install using yum on CI by adding a `yum_requirements.txt` file to the recipe folder.

**Comment by isuruf** [comment]
>  Is it also possible to only use yum_requirements.txt for one arch?

It's not at the moment.

> I will just take much longer and probably take too long for CI, so I decided to take it out for now. 

You can try cross-compiling from x86_64 to ppc64le

**Pull request merged**